Resources/config/publish-workflow.xml

```diff
@@ -29,34 +29,34 @@
             <argument>%cmf_core.publish_workflow.view_non_published_role%</argument>
         </service>
 
-        <service id="cmf_core.security.publishable_voter" class="%cmf_core.security.publishable_voter.class%">
+        <service id="cmf_core.security.publishable_voter" class="%cmf_core.security.publishable_voter.class%" public="false">
             <tag name="cmf_published_voter" priority="10"/>
         </service>
 
-        <service id="cmf_core.security.publish_time_period_voter" class="%cmf_core.security.publish_time_period_voter.class%">
+        <service id="cmf_core.security.publish_time_period_voter" class="%cmf_core.security.publish_time_period_voter.class%" public="false">
             <tag name="cmf_published_voter" priority="20"/>
         </service>
 
-        <service id="cmf_core.publish_workflow.request_listener" class="%cmf_core.publish_workflow.request_listener.class%">
+        <service id="cmf_core.publish_workflow.request_listener" class="%cmf_core.publish_workflow.request_listener.class%" public="false">
```

Comment:
i think this leads to 

[InvalidArgumentException]

The service "cmf_core.publish_workflow.request_listener" must be public as event subscribers are lazy-loaded.